src/main/java/Contexts/KeyWordContext.java

```diff
@@ -0,0 +1,18 @@
+package Contexts;
+
+import javafx.util.Pair;
+
+import java.util.ArrayList;
+
+public abstract class KeyWordContext {
+
+//    ArrayList<String> getPossibilities(String key) {
+//        return null;
+//    }
+//
+//    Pair<String, String> getSpellingCheck(String key);
+//    void AddKeyWord(String key);
+//    void clearContext();
+//    void removeKeyWords(String key);
```

Comment:
Why this chunk of commented out code? Remove them if not used.

Same for those below.